<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sysSetting</class>
 <widget class="QWidget" name="sysSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1061</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sysSetting</string>
  </property>
  <widget class="QLabel" name="backLightLab">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>背光亮度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="lightHorSlider">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>570</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(67,51, 139);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="sysTimeSetLab">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;系统时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addHourBut">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/moveup.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hourShowLab">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>63</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>63</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72,61, 139);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>13</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reduceHourBut">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/movedown.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addMinBut">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>330</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/moveup.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="minShowLab">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>400</y>
     <width>63</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>63</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72,61, 139);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>27</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reduceMinBut">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>460</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/movedown.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addSecondBut">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/moveup.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="secondShowLab">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>63</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>63</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72,61, 139);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reduceSecondBut">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/movedown.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addYearBut">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>103</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/moveup.bmp);
border-radius:5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="yearShowLab">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>171</y>
     <width>63</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>63</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72,61, 139);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;2020&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reduceYearBut">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/movedown.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addMonthBut">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>103</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/moveup.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="monthShowLab">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>171</y>
     <width>63</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>63</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72,61, 139);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reduceMonthBut">
   <property name="geometry">
    <rect>
     <x>339</x>
     <y>230</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/movedown.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addDayBut">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>103</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/moveup.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dayShowLab">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>171</y>
     <width>63</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>63</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(72,61, 139);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reduceDayBut">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>63</width>
     <height>63</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>51</width>
     <height>51</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/movedown.bmp);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sysLanguageLab">
   <property name="geometry">
    <rect>
     <x>762</x>
     <y>161</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>系统语言</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="sysLanguageListWid">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>210</y>
     <width>256</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="sysSettingBGLab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sysDateTextLab">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>178</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;日期&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sysYearTextLab">
   <property name="geometry">
    <rect>
     <x>237</x>
     <y>178</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;年&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sysMonthTextLab">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>178</y>
     <width>70</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;月&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sysDayTextLab">
   <property name="geometry">
    <rect>
     <x>576</x>
     <y>178</y>
     <width>70</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;日&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sysTimeTextLab">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>405</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>91</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;时间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sysTimeSignal1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sysTimeSignal2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(67,51, 139);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>sysSettingBGLab</zorder>
  <zorder>addHourBut</zorder>
  <zorder>hourShowLab</zorder>
  <zorder>reduceHourBut</zorder>
  <zorder>addMinBut</zorder>
  <zorder>minShowLab</zorder>
  <zorder>reduceMinBut</zorder>
  <zorder>addSecondBut</zorder>
  <zorder>secondShowLab</zorder>
  <zorder>reduceSecondBut</zorder>
  <zorder>addYearBut</zorder>
  <zorder>yearShowLab</zorder>
  <zorder>reduceYearBut</zorder>
  <zorder>addMonthBut</zorder>
  <zorder>monthShowLab</zorder>
  <zorder>reduceMonthBut</zorder>
  <zorder>addDayBut</zorder>
  <zorder>dayShowLab</zorder>
  <zorder>reduceDayBut</zorder>
  <zorder>sysLanguageLab</zorder>
  <zorder>backLightLab</zorder>
  <zorder>lightHorSlider</zorder>
  <zorder>sysTimeSetLab</zorder>
  <zorder>sysLanguageListWid</zorder>
  <zorder>sysDateTextLab</zorder>
  <zorder>sysYearTextLab</zorder>
  <zorder>sysMonthTextLab</zorder>
  <zorder>sysDayTextLab</zorder>
  <zorder>sysTimeTextLab</zorder>
  <zorder>sysTimeSignal1</zorder>
  <zorder>sysTimeSignal2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
